app/controllers/concerns/curation_concerns/file_sets_controller_behavior.rb

```diff
@@ -191,7 +191,7 @@ def json_error(error, name = nil, additional_arguments = {})
       end
 
       def empty_file?(file)
-        (file.respond_to?(:tempfile) && file.tempfile.empty?) || (file.respond_to?(:size) && file.empty?)
+        (file.respond_to?(:tempfile) && file.tempfile.blank?) || (file.respond_to?(:size) && file.blank?)
```

Comment:
@awead does this make sense? I was getting some `NoMethodErrors` for `#empty?`

Reply:
@RudyOnRails yes, I often use `present?` and `blank?` because they handle arrays that are either `[]` or `nil`.